runtime-modules/working-group/src/tests/mod.rs

```diff
@@ -1592,8 +1617,37 @@ fn set_working_group_mint_capacity_fails_with_mint_not_found() {
 fn set_working_group_mint_capacity_fails_with_invalid_origin() {
     build_test_externalities().execute_with(|| {
         let capacity = 15000;
-        let result = WorkingGroup1::set_mint_capacity(RawOrigin::None.into(), capacity);
+        let result = TestWorkingGroup::set_mint_capacity(RawOrigin::None.into(), capacity);
 
         assert_eq!(result, Err(Error::RequireRootOrigin));
     });
 }
+
+#[test]
+fn ensure_setting_genesis_storage_working_group_mint_succeeds() {
```

Comment:
`_storage` in the names of the test for the working group can be dropped.

Reply:
Fixed.